.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve Streamlink
+
```

Comment:
Are these empty lines required? Just curious :)

Reply:
I don't think they are, but the only example I can find that uses them for the issue templates have them.